sdk/core/System.ClientModel/src/Pipeline/ClientRetryPolicy.cs

```diff
@@ -133,10 +133,10 @@ private async ValueTask ProcessSyncOrAsync(PipelineMessage message, IReadOnlyLis
 
     protected virtual void OnTryComplete(PipelineMessage message) { }
 
-    public bool ShouldRetry(PipelineMessage message, Exception? exception)
+    internal bool ShouldRetry(PipelineMessage message, Exception? exception)
```

Comment:
I think it can be protected, can't it?

Reply:
Yep, if we can remove the Core suffix, we can make these protected virtual.  Updating...